### PR TITLE
UX and text improvements

### PR DESCRIPTION
**Pull request by 0xmiel**
- [x] Fix https://github.com/1Hive/gardens/issues/258
- [x] Fix https://github.com/1Hive/gardens/issues/197
- [x] Rename Collateral manager to deposit manager and simplify explanations (in order to ensure consistency with the above fix)
- [x] Fix https://github.com/1Hive/gardens/issues/256
- [x] Fix https://github.com/1Hive/gardens/issues/255
- [x] Fix https://github.com/1Hive/gardens/issues/243
- [x] Fix https://github.com/1Hive/gardens/issues/254
- [x] Fix https://github.com/1Hive/gardens/issues/265

### CODE REVIEW

File: src/components/Garden/Stake/BalanceCard.js [hunk 1]
```diff
@@ -141,8 +145,8 @@ function BalanceCard({
         >
           <Help hint="">
             {allowManagerDisabled
-              ? `You cannot disallow this garden from locking your funds at this time because you have some collateral locked in scheduled proposals. Once the proposals are finalised you’ll be able to disallow the garden.`
-              : `By enabling this feature you allow this garden to lock the available ${tokenSymbol} you have deposited when creating actions.`}
+              ? `You cannot revoke the deposit manager's ability to lock your funds at this time. You need to wait until your open proposals (proposals you have either created or challenged) have either been completed or removed - scroll to the bottom of the page to see which proposals are concerned.`
+              : `By enabling this feature you allow the deposit manager to lock ${tokenSymbol} funds you have deposited on any proposal that you create or challenge. You need to enable this feature to be able to create or challenge a proposal.`}
```
Comment (0xGabi) [inline src/components/Garden/Stake/BalanceCard.js:149]:
```suggestion
              : `By enabling this feature you give permission to the deposit manager to lock ${tokenSymbol} funds you have deposited on any proposal that you create or challenge. You need to enable this feature to be able to create or challenge a proposal.`}
```

File: src/components/Garden/Stake/BalanceCard.js [hunk 2]
```diff
@@ -132,7 +136,7 @@ function BalanceCard({
             font-weight: 600;
           `}
         >
-          Allow Garden
+          Give Permissions
```
Comment (0xGabi) [inline src/components/Garden/Stake/BalanceCard.js:139]:
```suggestion
          Give Permission

File: src/hooks/useStaking.js [hunk 3]
```diff
@@ -360,7 +360,7 @@ export function useStaking() {
             data: allowManagerData,
             from: account,
             to: stakeManagement.stakingInstance,
-            description: 'Allow Garden',
+            description: 'Give Permissions',
```
Comment (0xGabi) [inline src/hooks/useStaking.js:363]:
```suggestion
            description: 'Give Permission',

File: src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js [hunk 4]
```diff
@@ -1,16 +1,39 @@
 import React, { useMemo, useState, useCallback } from 'react'
+import { useHistory } from 'react-router'
+import { Button } from '@1hive/1hive-ui'
 import ModalFlowBase from '../ModalFlowBase'
 import ChangeSupport from './ChangeSupport'
 import SupportProposal from './SupportProposal'
 
+// import { buildGardenPath } from '@utils/routing-utils'
```
Comment (0xGabi) [inline src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js:8]: We should remember to remove this line.

File: src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js [hunk 5]
```diff
@@ -1,16 +1,39 @@
...
+          label="View supported proposals"
+          mode="strong"
+          onClick={handleViewProposal}
+          wide
+        />
+      )
+    }
+    return null
+  }, [handleViewProposal, mode])
+
```
Comment (0xGabi) [inline src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js:36]: I am not convinced by this behavior. What do you think of instead change to the main view of the specific Garden?

Comment (0xGabi) [reply src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js:36]: What you think of skip this change from the current PR and instead create a separate one to address it?

**Review by 0xGabi — CHANGES_REQUESTED** [review]
A lot of awesome changes to the UX 💅 I like much more the new naming `Deposit Amount`.

I left several suggestions, and I think the current changes have a lot of value to be included after we address the suggestions.

There are a few bits of logic that may need further consideration (proposal creation modal and support proposal handler). As I mentioned in the review we can address them in separate PRs 👍

Well done!

File: src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js [hunk 6]
```diff
@@ -205,6 +205,18 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   return (
     <form onSubmit={handleOnContinue}>
+      <Info title="Proposal guidelines">
+        All proposals are bound by this community's covenant - if you haven't
+        taken the time to read through it yet, please make sure you do so.
```
[the remark this answers is not in the document]
Comment (0xmiel) [reply src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js:210]: Good call, meant to go back to this to do exactly that.

File: src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js (hunk 5 above)
Comment (0xmiel) [reply src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js:36]: Agree, this is not ideal.

So I think the natural tendency is to look for confirmation that the support has indeed been placed, with the amount specified.

The home page doesn't quite achieve that.

My initial solution was to simply refresh the page, but unfortunately that disconnects the account.

Ideally, one would just make the modal disappear.

File: src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (0xmiel) [reply src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js:210]: Will start on this tomorrow

File: src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js [hunk 7]
```diff
@@ -205,6 +205,18 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   return (
     <form onSubmit={handleOnContinue}>
+      <Info title="Proposal guidelines">
+        All proposals are bound by this community's covenant - if you haven't
```
[the remark this answers is not in the document]
Comment (0xmiel) [reply src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js:209]: Good call, meant to go back to do exactly this

File: src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js (hunk 5 above)
Comment (0xGabi) [reply src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js:35]: Yes, maybe we can make the modal disappear and integrate something like [blocknative's notify](https://docs.blocknative.com/notify) showing a toast at the bottom of the page.

Comment (0xmiel) [reply src/components/Garden/ModalFlows/SupportProposal/SupportProposalScreens.js]: Ok, removing so we can merge. will address in another PR

**Pull request merged**